include/data/items.h

```diff
@@ -1,6460 +1,6079 @@
-#ifndef GUARD_DATA_ITEMS
-#define GUARD_DATA_ITEMS
-
-const struct Item gItems[] = {
-
-		[ITEM_NONE] = {
-		.name = _("????????$"),
-		.itemId = ITEM_NONE,
-		.price = 0,
-		.holdEffect = 0,
-		.holdEffectParam = 0,
-		.description = gDummyItemDescription,
-		.importance = 0,
-		.unk19 = 0,
-		.pocket = POCKET_ITEMS,
-		.type = 4,
-		.fieldUseFunc = ItemUseOutOfBattle_CannotUse,
-		.battleUsage = 0,
-		.battleUseFunc = NULL,
-		.secondaryId = 0,
-	},
-	
-	//Pokeballs
-	
-	[ITEM_MASTER_BALL] = {
-		.name = _("MASTER BALL$"),
```

Comment:
```
[21:15] ProjectRevoTPP: dont add a $
[21:15] ProjectRevoTPP: the string builder does that
[21:15] ProjectRevoTPP: const u8 myString[] = _("Foo");
```

Did this turn out not to be the case?

Reply:
```
+//Pokeballs
+
+    {
+        .name = _("MASTER BALL"),
```

I don't see a $.

Reply:
Disregard this I'm an idiot, I got confused between remove and add.